.github/workflows/run-tests.yaml

```diff
@@ -17,6 +17,8 @@ jobs:
           distribution: 'temurin'
       - name: Build with Maven, run unit tests and the coverage check
         run: mvn clean install
+      - name: Build Examples
```

Comment:
Can we avoid building docker images? The executions inside the examples pom will build the docker images, which can take longer.